syntax = "proto2";

package cal;

message CalReqHeader {
  required fixed32 msg_len = 1;
};

message CalReq {
  required string seqno = 1; 
  optional string optr = 2;
  repeated int32 opnd_arr = 3 [packed=true];
};
